src/openms/include/OpenMS/KERNEL/MSExperiment.h

```diff
@@ -697,6 +698,22 @@ namespace OpenMS
       return *this;
     }
 
+    /// get the file path to the first MS run
+    StringList getPrimaryMSRunPath() const
+    {
+      StringList ms_run_paths;
+      std::vector<SourceFile> sfs(this->getSourceFiles());
+      for (std::vector<SourceFile>::const_iterator it = sfs.begin(); it != sfs.end(); ++it)
+      {
+        String ms_run_location = it->getPathToFile() + "/" + it->getNameOfFile();
```

Comment:
Maybe add a comment that URIs are used and thus "/" is fine for every platform, for the next person wondering this?